src/wrapper/objects/jlist.rs

```diff
@@ -52,6 +52,8 @@ impl<'a: 'b, 'b> JList<'a, 'b> {
         let remove = env.get_method_id(class, "remove", "(I)Ljava/lang/Object;")?;
         let size = env.get_method_id(class, "size", "()I")?;
 
+        let _ = env.delete_local_ref(class.into());
```

Comment:
I'm not sure I understand. Should I do `let _ = env.auto_local(class.into());` here?

Reply:
I suggest not here, but right after the class initialization. The lookups will just have to replace `class` with `&class` I believe.

Reply:
I changed the initialization line. I wasn't sure if I should do it in two lines instead to improve readability. So it would first call `find_class` and the create the `AutoLocal` in a separate line.

Reply:
I think both are good, thanks!